index.js

```diff
@@ -74,7 +74,7 @@ LiveServer.start = function(port, directory, suppressBrowserLaunch) {
 		.use(staticServer(directory)) // Custom static server
 		.use(connect.directory(directory, { icons: true }))
 		.use(connect.logger('dev'));
-	var server = http.createServer(app).listen(port);
+	var server = http.createServer(app).listen(port, '0.0.0.0');
```

Comment:
According to [node docs](http://nodejs.org/api/http.html#http_server_listen_port_hostname_backlog_callback), without the parameter it listens to all interfaces (which I think equals to 0.0.0.0) so is this strictly needed, or is it just the address you type to the browser that makes the difference? (Trying to understand the exact issue in order to not break it in the future).